include/SFML/Graphics/Color.inl

```diff
@@ -22,31 +22,23 @@
 //
 ////////////////////////////////////////////////////////////
 
-////////////////////////////////////////////////////////////
-// Headers
-////////////////////////////////////////////////////////////
-#include <SFML/Graphics/Color.hpp>
-#include <algorithm>
-
 
-namespace sf
-{
 ////////////////////////////////////////////////////////////
 // Static member data
 ////////////////////////////////////////////////////////////
-const Color Color::Black(0, 0, 0);
-const Color Color::White(255, 255, 255);
-const Color Color::Red(255, 0, 0);
-const Color Color::Green(0, 255, 0);
-const Color Color::Blue(0, 0, 255);
-const Color Color::Yellow(255, 255, 0);
-const Color Color::Magenta(255, 0, 255);
-const Color Color::Cyan(0, 255, 255);
-const Color Color::Transparent(0, 0, 0, 0);
+inline const Color Color::Black{0, 0, 0};          //!< Black predefined color
+inline const Color Color::White{255, 255, 255};    //!< White predefined color
+inline const Color Color::Red{255, 0, 0};          //!< Red predefined color
+inline const Color Color::Green{0, 255, 0};        //!< Green predefined color
+inline const Color Color::Blue{0, 0, 255};         //!< Blue predefined color
+inline const Color Color::Yellow{255, 255, 0};     //!< Yellow predefined color
+inline const Color Color::Magenta{255, 0, 255};    //!< Magenta predefined color
+inline const Color Color::Cyan{0, 255, 255};       //!< Cyan predefined color
+inline const Color Color::Transparent{0, 0, 0, 0}; //!< Transparent (black) predefined color
 
 
 ////////////////////////////////////////////////////////////
-Color::Color() :
+constexpr inline Color::Color() :
```

Comment:
Depending on outcome [of this discussion](https://github.com/SFML/SFML/pull/1903#discussion_r769076008), only `constexpr`